cargo-miri/src/util.rs

```diff
@@ -249,3 +251,26 @@ pub fn debug_cmd(prefix: &str, verbose: usize, cmd: &Command) {
     }
     eprintln!("{prefix} running command: {cmd:?}");
 }
+
+/// Determines where the sysroot of this exeuction is
+/// 
+/// Either in a user-specified spot by an envar, or in a default cache location.
+pub fn get_sysroot() -> PathBuf {
+    match std::env::var_os("MIRI_SYSROOT") {
+        Some(dir) => PathBuf::from(dir),
+        None => {
+            let user_dirs = directories::ProjectDirs::from("org", "rust-lang", "miri").unwrap();
+            user_dirs.cache_dir().to_owned()
+        }
+    }
+}
+
+/// Deletes the Miri sysroot cache directory.
+pub fn clean_sysroot() {
+    let sysroot_dir = get_sysroot();
+
+    eprint!("[cargo-miri] clearing cache at {}", sysroot_dir.display());
```

Comment:
Oh yeah I'll switch that. I chose eprint[ln]!() because it is used elsewhere in cargo-miri for outputs.

https://github.com/rust-lang/miri/blob/61af95673d0d9df50052c43e4ae2302824ef1089/cargo-miri/src/util.rs#L191
https://github.com/rust-lang/miri/blob/61af95673d0d9df50052c43e4ae2302824ef1089/cargo-miri/src/util.rs#L250
https://github.com/rust-lang/miri/blob/61af95673d0d9df50052c43e4ae2302824ef1089/cargo-miri/src/phases.rs#L299-L306
and others. Unless I misunderstood what those are for. But I'll switch it to eprintln!()

Reply:
You'll notice that every single one of them is under an `if` that makes it only show in "verbose" mode.

Reply:
I do not believe that is true for all of them, mainly in setup.rs

https://github.com/rust-lang/miri/blob/61af95673d0d9df50052c43e4ae2302824ef1089/cargo-miri/src/setup.rs#L147-L157
https://github.com/rust-lang/miri/blob/61af95673d0d9df50052c43e4ae2302824ef1089/cargo-miri/src/setup.rs#L177-L187

But regardless, it doesn't make a difference to me. Should these be println!() instead?

Reply:
Ah true, those are not "verbose" mode.

It seems like cargo prints its stuff to stderr as well, so `eprintln!` should be fine. The `[cargo-miri rustc]` prefix makes no sense here though, that's indicating where in cargo-miri certain verbose debug output comes from.